editor/components/block-settings-menu/index.js

```diff
@@ -83,7 +84,7 @@ export class BlockSettingsMenu extends Component {
 									onToggle();
 								} }
 								icon="ellipsis"
-								label={ __( 'More Options' ) }
+								label={ __( label ) }
```

Comment:
You can't pass variables to the i18n function. Should be `const label = isOpen ? __( 'Hide Options' ) : __( 'More Options' );` in line 75.

Reply:
Updated